client/src/components/AuthForm/index.tsx

```diff
@@ -82,25 +81,6 @@ const AuthForm: React.FC = () => {
           Please enter your details here to continue...
         </Typography>
       </Stack>
-      {!loginMode && (
-        <Stack>
-          <Typography variant="body2">Username</Typography>
-          <Controller
-            control={control}
-            name="username"
-            defaultValue=""
-            rules={{ required: 'Username is required' }}
-            render={({ field }) => (
-              <StyledTextField
-                {...field}
-                placeholder="Enter your username"
-                error={Boolean(errors.username)}
-                helperText={errors.username && errors.username.message}
-              />
-            )}
-          />
-        </Stack>
-      )}
```

Comment:
Why we removed this piece of code ?

Reply:
I don't think we require a username, an email-id should be enough

Reply:
Oh okay cool!